src/current/releases/molt.md

```diff
@@ -0,0 +1,155 @@
+---
+title: MOLT Releases
+summary: Changelog for MOLT Fetch, Verify, and Live Migration Service
+toc: true
+docs_area: releases
+---
+
+This page has details about each release of the following [MOLT (Migrate Off Legacy Technology) tools]({% link molt/molt-overview.md %}):
+
+- [Fetch]({% link molt/molt-fetch.md %}) and [Verify]({% link molt/molt-verify.md %})
+- [Live Migration Service]({% link molt/live-migration-service.md %})
+
+Cockroach Labs recommends using the latest available version of each tool. See [Installation](#installation).
+
+<div class="filters filters-big clearfix">
+    <button class="filter-button" data-scope="fetch_verify">Fetch/Verify</button>
+    <button class="filter-button" data-scope="lms">LMS</button>
+</div>
+
+## Installation
+
+<section class="filter-content" markdown="1" data-scope="fetch_verify">
+To download the latest MOLT Fetch/Verify binary:
+
+{% include molt/molt-install.md %}
+
+{{site.data.alerts.callout_info}}
+[MOLT Fetch]({% link molt/molt-fetch.md %}) is supported on Red Hat Enterprise Linux (RHEL) 9 and above.
+{{site.data.alerts.end}}
+</section>
+
+<section class="filter-content" markdown="1" data-scope="lms">
+To install the LMS, refer to [MOLT Live Migration Service]({% link molt/live-migration-service.md %}#installation).
+
+To update the LMS once installed, configure the following [Helm chart values](https://github.com/cockroachdb/molt-helm-charts/blob/main/lms/values.yaml):
+
+{% include molt/update-lms-version.md %}
+
+For more information, refer to [Configuration]({% link molt/live-migration-service.md %}#configuration).
+</section>
+
+<section class="filter-content" markdown="1" data-scope="fetch_verify">
+## July 10, 2024
+
+MOLT Fetch/Verify 1.1.1 is [available](#installation).
+
+- Fixed a bug that led to incorrect list continuation file behavior if a trailing slash was provided in `--bucket-path`.
+- Fixed a bug with extracting the filename from a failed import URL. Previously, an older filename was being used, which could result in duplicated data. Now, the filename that is used in import matches what is stored in the exceptions log table.
+- Added a `--use-implicit-auth` flag that determines whether [implicit authentication]({% link {{ site.current_cloud_version }}/cloud-storage-authentication.md %}) is used for cloud storage import URIs.
+
+## July 8, 2024
+
+MOLT Fetch/Verify 1.1.0 is [available](#installation).
+
+- Fixed a bug where integer values for `JSONB`/`JSON` types were not correctly handled when migrating from a PostgreSQL or CockroachDB source.
+- Schemas can now be created from Oracle sources.
```

Comment:
Do we want to publish this now, considering that the data migration part doesn't work yet?

Reply:
Yep, I'll just remove the note.